2022q3/cloud-init.adoc

```diff
@@ -0,0 +1,21 @@
+=== FreeBSD as a Tier 1 cloud-init Platform
+
+Links: +
+link:https://https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst] +
+
+Contact: Mina Galić <me+FreeBSD@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Unfortunately, cloud-init’s support for operating systems other than Linux is rather poor and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier 1 platform.
+To remedy this, this project aims to bring FreeBSD cloud-init support on par with Linux support. Though the plan is to lift support across all BSDs.
```

Comment:
```suggestion
To remedy this, this project aims to bring FreeBSD cloud-init support on par with Linux support. However, the plan is to lift support across all BSDs.
```

Reply:
@pauamma sorry, I overlooked this when I made <https://github.com/igalic/freebsd-quarterly/pull/1>.